include/pokemon.h

```diff
@@ -445,15 +445,15 @@ struct SpeciesInfo /*0x8C*/
             // Move Data
  /* 0x80 */ const struct LevelUpMove *levelUpLearnset;
  /* 0x84 */ const u16 *teachableLearnset;
-            const u16 *eggMoveLearnset;
- /* 0x88 */ const struct Evolution *evolutions;
- /* 0x84 */ const u16 *formSpeciesIdTable;
- /* 0x84 */ const struct FormChange *formChangeTable;
+ /* 0x88 */ const u16 *eggMoveLearnset;
+ /* 0x8C */ const struct Evolution *evolutions;
+ /* 0x90 */ const u16 *formSpeciesIdTable;
+ /* 0x94 */ const struct FormChange *formChangeTable;
```

Comment:
Why does it go from 0x94 to 0xB4? It should be
```
/* 0x98 */ struct ObjectEventGraphicsInfo overworldData;
```
Not sure what's the length of the `ObjectEventGraphicsInfo`, but the next entry should be `0x98 + struct size`

Reply:
the size is 20 bytes. should be fixed